test/e2e/singlecluster/pod_test.go

```diff
@@ -80,8 +80,8 @@ var _ = ginkgo.Describe("Pod groups", func() {
 		})
 
 		ginkgo.It("should admit group that fits", func() {
-			group := podtesting.MakePod("group", ns.Name).
-				Image(util.GetAgnHostImage(), util.BehaviorExitFast).
```

Comment:
why this image needs to change?

Reply:
I'm not entirely sure here.

I don't have much context into what upstream did here.

https://github.com/kubernetes-sigs/kueue/blob/main/test/util/e2e.go#L66

Reply:
https://github.com/kubernetes-sigs/kueue/pull/5780

Reading that I was thinking that we may hit problems using this image and I had to rebase this so I picked the old image for now.